apps/room/templatetags/room_tags.py

```
@@ -30,3 +30,97 @@ def room_url(parser, token):
+        integer_part = thousand_sep.join(grouped)[::-1]
+        
+        if is_negative:
+            integer_part = '-' + integer_part
+        
+        # Reconstruct the number
+        if decimal_part:
+            result = f"{integer_part}{decimal_sep}{decimal_part}"
+        else:
+            result = integer_part
+        
+        return mark_safe(result)
```

Ruff (0.15.12)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)